<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1496</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_first">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ready">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>690</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready ?</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>591</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartDistance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>861</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="ChartSignal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>861</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1496</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelo">
    <property name="title">
     <string>Helo</string>
    </property>
   </widget>
   <addaction name="menuHelo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
